EpiAware/test/EpiLatentModels/manipulators/broadcast/LatentModel.jl

```diff
@@ -22,7 +22,7 @@ end
 @testitem "generate_latent function with BroadcastLatentModel" begin
     using Turing, DynamicPPL
     model = BroadcastLatentModel(RandomWalk(), 5, RepeatBlock())
-    broadcasted_model = generate_latent(model, 10)
+    broadcasted_model = generate_latent(model, 15)
```

Comment:
Just checking, but is this just because epsilon_t is now `n-1` length?

Reply:
yes exactly